packages/react-components/react-tags/src/components/Tag/Tag.test.tsx

```diff
@@ -20,8 +20,14 @@ describe('Tag', () => {
     requiredProps,
   });
 
-  it('should render root as a button', () => {
-    const { getByRole } = render(<Tag>Tag</Tag>);
+  it('should render root as a span', () => {
+    const { getByTestId } = render(<Tag data-testid="testid">Tag</Tag>);
+    expect(getByTestId('testid').tagName).toBe('SPAN');
+    expect(getByTestId('testid').className.includes(tagClassNames.root)).toBe(true);
+  });
+
+  it('should render root as a button for dismissible tag', () => {
+    const { getByRole } = render(<Tag dismissible>Tag</Tag>);
```

Comment:
```ts
const { queryByRole } = render(<Tag dismissible>Tag</Tag>);
expect(queryByRole('button')).not.toBe(null);
```

nit: this should be enough since in the test you _**know**_ you're not rendering anything else